src/mame/namco/baraduke.cpp

```diff
@@ -110,7 +110,9 @@ DIP locations verified for:
 #include "cpu/m6800/m6801.h"
 #include "cpu/m6809/m6809.h"
 #include "machine/watchdog.h"
+#include "namco_cus4xtmap.h"
 #include "sound/namco.h"
+#include "video/resnet.h"
```

Comment:
Local project headers (in this case, `"namco_cus4xtmap.h"`) should be included before headers from src/devices.